BraveRewards/Ledger/Data/BATLedgerDatabase.mm

```diff
@@ -9,13 +9,24 @@
 
 @implementation BATLedgerDatabase
 
-+ (nullable __kindof NSManagedObject *)firstOfClass:(Class)clazz withPublisherID:(NSString *)publisherID context:(NSManagedObjectContext *)context
++ (nullable __kindof NSManagedObject *)firstOfClass:(Class)clazz
+                                    withPublisherID:(NSString *)publisherID
+                                    customPredicate:(nullable NSPredicate *)customPredicate
```

Comment:
`customPredicate` → `additionalPredicate`, since its not replacing the `publisherID` predicate :)